.gitignore

```diff
@@ -10,68 +9,63 @@ cabal-dev
 *.aux
 *.hp
 *.tix
-.virtualenv
-.hsenv
-.hpc
-.Werror
-*.svg
-.stack-work/
+
+# Cabal
+cabal-dev
 .cabal-sandbox/
 cabal.sandbox.config
 cabal.config
-GTAGS
-GRTAGS
-GPATH
-TAGS
```

Comment:
Why did you delete e. g. `TAGS`? IIRC there is some software which dumps this file and perhaps some people use it.